fabric-rendering-v1/src/main/java/net/fabricmc/fabric/mixin/client/rendering/MixinInGameHud.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.client.rendering;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import net.minecraft.client.gui.hud.InGameHud;
+
+import net.fabricmc.fabric.api.client.rendering.v1.HudRenderCallback;
+
+@Mixin(InGameHud.class)
+public class MixinInGameHud {
+	@Inject(method = "render", at = @At(value = "RETURN", shift = At.Shift.BY, by = -6))
```

Comment:
This shift is fragile, and a bit of a pain to see where its injected to. Is there a reson why you didnt inject it at the specific instruction?

Reply:
I wanted to inject before `color4f` but there is 4 `color4f`.
I can inject into `enableAlphaTest` and shift by -3 but there is 3 `enableAlphaTest`.
Or I guess we can inject to `RETURN` and call
```java
RenderSystem.color4f(1.0F, 1.0F, 1.0F, 1.0F);
RenderSystem.enableAlphaTest();
```
ourselves to reset the stuff.

Reply:
Ok, this is fine then. The worst I can see happening is it moving and things rendering weirdly.